### ISSUE TITLE
DEV VM updated to V27, TRUDUtils installed but not working

### ISSUE THREAD

**Issue by jeroeneikmans** [TrudAX/TRUDUtilsD365#13]
Hi, I've updated a DEV VM to V27. After that I installed TRUDUtils, but no menu extensions are shown. It looks like the TRUDUtils is not loaded when VS is started. How can I fix this?

**Comment by jeroeneikmans** [comment]
I figured this one out. After V27 update it seems that the environment variable DynamicsVSTools is not right anymore. At least on my machine. Fixed this by manually changing to the right path.

**Comment by jeroeneikmans** [comment]
I did run InstallToVS again, but for some reason the path to DynamicsVSTools was not ok after updating the VM to V27. I fixed it by changing the windows environment variable 'DynamicsVSTools'. I looked for the file 'Microsoft.Dynamics.Framework.Tools.AddIns.Common.dll' in the VS2019 program files path to find the right folder. Also, maybe it has something to do with having VS2017 and VS2019 installed on the VM. The use of the environment variable 'DynamicsVSTools' is not reliable then.
